spec/ivapi/client/account_spec.rb

```diff
@@ -24,6 +24,18 @@
     end
   end
 
+  describe 'account credits' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.